common/src/main/java/com/wynntils/models/raid/RaidModel.java

```diff
@@ -39,13 +42,18 @@ public class RaidModel extends Model {
     private static final Pattern RAID_COMPLETED_PATTERN = Pattern.compile("§f§lR§#4d4d4dff§laid Completed!");
     private static final Pattern RAID_FAILED_PATTERN = Pattern.compile("§4§kRa§c§lid Failed!");
 
+    private static final Pattern RAID_CHOOSE_BUFF_PATTERN = Pattern.compile(
+            "§#d6401effó\u008F¿¼î€\u0081ó\u0090€† §(#fa7f63ff|e)(\\w+)§#d6401eff has chosen the §#fa7f63ff(\\w+ \\w+)§#d6401eff buff!");
```

Comment:
If you change the pattern to `§#d6401eff(\uE009\uE002|\uE001) §#fa7f63ff(\w+)§#d6401eff has chosen the §#fa7f63ff(\w+ \w+)§#d6401eff buff!` then you can use `event.getOriginalStyledText().stripAlignment().getMatcher(RAID_CHOOSE_BUFF_PATTERN)` in the event to make it a little cleaner.

Reply:
Also I will note here this pattern is not perfect, as nicknamed members will have the `§o` before their name and you will have to get their actual username from the hover event, see `RevealNicknamesFeature` on how you can do that